_posts/2023-01-27-PC-Engines_Firmware-v4-19-0-1.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title:  "PC Engines apu coreboot Open Source Firmware v4.19.0.1"
+date:   2023-02-02
+categories: Firmware
+---
+# PC Engines apu coreboot Open Source Firmware v4.19.0.1
+
+## Key changes
+
+Mainline:
+
+### Changed
+
+- **Rebased** with official coreboot repository **commit** 8d3e714
+- Removed configuration and mainboard files for apu1
```

Comment:
PLease use the bulletpoint from coreboot's CHANGELOG.md file https://github.com/pcengines/coreboot/blob/release/CHANGELOG.md?plain=1#L19

Reply:
improved